static/app/components/radio.tsx

```diff
@@ -12,8 +12,8 @@ type CheckedProps = Props &
 
 const checkedCss = (p: CheckedProps) => css`
   display: block;
-  width: ${p.radioSize === 'small' ? '8px' : '1rem'};
-  height: ${p.radioSize === 'small' ? '8px' : '1rem'};
+  width: ${p.radioSize === 'small' ? '8px' : '0.625rem'};
+  height: ${p.radioSize === 'small' ? '8px' : '0.625rem'};
```

Comment:
Scaling down by a factor of `10/16 = 0.625`.

Reply:
Also, the use of different units (`px` vs `rem`) here is kinda strange. Once this PR gets merged, the units will all be changed to `rem` in https://github.com/getsentry/sentry/pull/30697.